features/all-positions/AllPositions.tsx

```diff
@@ -76,84 +128,189 @@ const AllPositions = () => {
-                <TableHead>
-                  <TableRow>
-                    <TableCell>Sponsor</TableCell>
-                    <TableCell align="right">
-                      Collateral
-                      <br />({collSymbol})
-                    </TableCell>
-                    <TableCell align="right">
-                      Synthetics
-                      <br />({tokenSymbol})
-                    </TableCell>
-                    <TableCell align="right">Collateral Ratio</TableCell>
-                    <Tooltip
-                      title={`This is the price that the identifier (${priceIdUtf8}) must increase to in order for the position be liquidatable`}
-                      placement="top"
-                    >
-                      <TableCell align="right">Liquidation Price</TableCell>
-                    </Tooltip>
-                  </TableRow>
-                </TableHead>
-                <TableBody>
-                  {Object.keys(activeSponsors).map((sponsor: string) => {
-                    const activeSponsor = activeSponsors[sponsor];
-                    return (
-                      activeSponsor?.collateral &&
-                      activeSponsor?.tokensOutstanding && (
-                        <TableRow key={sponsor}>
-                          <TableCell component="th" scope="row">
-                            <a href={getEtherscanUrl(sponsor)} target="_blank">
-                              {prettyAddress(sponsor)}
-                            </a>
-                          </TableCell>
-                          <TableCell align="right">
-                            {prettyBalance(Number(activeSponsor.collateral))}
-                          </TableCell>
-                          <TableCell align="right">
-                            {prettyBalance(
-                              Number(activeSponsor.tokensOutstanding)
-                            )}
-                          </TableCell>
-                          <TableCell align="right">
-                            {prettyBalance(
-                              getCollateralRatio(
-                                Number(activeSponsor.collateral),
+        {activeSponsors && Object.keys(activeSponsors).length > 0 && (
+          <>
+            <Box>
+              <Typography>
+                {`Estimated price of ${latestPrice} for ${priceIdUtf8} sourced from: `}
+                <Link
+                  href={sourceUrl}
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  Coinbase Pro.
+                </Link>
+              </Typography>
+            </Box>
+            <Box pt={4}>
+              <TableContainer component={Paper}>
+                <Table>
+                  <TableHead>
+                    <TableRow>
+                      <TableCell>Sponsor</TableCell>
+                      <TableCell align="right">
+                        <ClickableText
```

Comment:
Nothing changed here except adding the `ClickableText` wrappers around the column headers to enable sorting.